### DISCUSSION TITLE
create 2 projects in same account both using the same repository

### DISCUSSION THREAD

**Discussion by omar-chaabouni** [codemagic-ci-cd/codemagic-docs#1905] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I want to create another application with a repository that is already used in another application in the same codemagic account
I couldn't find the repository in repos list

### Expected results

finding the repo so I can create another app

### Actual results

repository not found

**Comment by dtrdic** [comment]
Hello @omar-chaabouni, try adding the app using the "Other" option. Then enter the Git URL for cloning your project
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/c0b2aaf6-0c09-4dc6-8a05-3015b2f38f08)

**Reply by omar-chaabouni** [reply]
Thank you for your response @dtrdic . Tried it, looks like it only supportS a git URL, doesn't support a bitbucket URL.

**Reply by dtrdic** [reply, marked as answer]
it supports bitbucket url as well. 
1. When cloning the repo don't use the git clone command, just the URL like this `git@bitbucket.org:test1/testapp.git`
2. Generate key pair and add the generated public key to your repository before adding the app
3. Go into your bitbucket repository settings > security > Access keys
4. Click on `Add key` and paste the content of the generated public key in there

**Reply by omar-chaabouni** [reply]
Thank you very David @dtrdic much appreciated!
